recipes/haystack-ai/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "haystack-ai" %}
+{% set version = "2.0.0b5" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@ocefpaf I tried to follow the instructions from the cfep, does it look ok now? Final release is planned for late Q1, it would be nice to have the beta package in the meantime.

Reply:
Remember to remove that file when you publish the final release on conda-forge.